app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -247,6 +308,27 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
 
     getHostname: function () {
       return window.location.hostname
+    },
+
+    getProtocol: function () {
+      return window.location.protocol
+    },
+
+    removeCrossDomainParams: function (clickData) {
+      var url = clickData.url
+      if (url.indexOf('_ga') !== -1 || url.indexOf('_gl') !== -1) {
+        // _ga & _gl are values needed for cross domain tracking, but we don't want them included in our click tracking.
+        url = url.replace(/_ga=([^&]*)/, '')
+        url = url.replace(/_gl=([^&]*)/, '')
```

Comment:
I think it might be possible to combine these two lines into a single regex: `_g[al]=([^&]*)`